docs/source/gbd2020_models/risk_attributable_causes/wasting/index.rst

```diff
@@ -591,7 +591,7 @@ that follows we will detail how to calculate all the variables used
      - Daily probability of remission from cat 2 into cat 3 (treated or untreated)
      - Nicole's calculations (get lit source!)
    * - r4
-     - 6-12 months: 0.000018 (90% UI: 0.000003, 0.000033); 1-4 years: 0.000014 (90% UI: 0.000003, 0.000046)
+     - 1 - e^(rate). 6-12 months: rate = 0.006140 (SD: 0.003015, normal distribution of uncertainty). 1-4 years: rate = 0.005043  (SD: 0.002428, normal distribution of uncertainty)
```

Comment:
SD is pretty large compared to mean, better truncate so that rate >= 0, and 1-e^-rate is non-negative
```suggestion
     - 1 - e^{-rate}. 6-12 months: rate = 0.006140 (SD: 0.003015, normal distribution of uncertainty). 1-4 years: rate = 0.005043  (SD: 0.002428, normal distribution of uncertainty)
```